engine/filters.go

```diff
@@ -77,6 +77,71 @@ func (fS *FilterS) Pass(tenant string, filterIDs []string,
 	return
 }
 
+//verifyPrefixes verify the Element and the Values if has as prefix one of the prefixes
+func verifyPrefixes(rule *FilterRule, prefixes []string) (hasPrefix bool) {
```

Comment:
Need review together, the following ideas came into my mind:
* The function can be part of the filter object.
* Not sure if additional if for dynamic is needed.
* Can send the prefixes as set instead of slice.
* The use of too many hasPrefix looks a bit odd at least.

As said, need to review it together.